packages/cosmic-swingset/lib/ag-solo/vats/lib-wallet.js

```diff
@@ -1,22 +1,30 @@
 import harden from '@agoric/harden';
 import { insist } from '@agoric/ertp/util/insist';
 
+import { makeUnitOps } from '@agoric/ertp/core/unitOps';
+
+import makeObservablePurse from './observable';
+import makeStore from './store';
+import makeWeakStore from './weakStore';
+
 function mockStateChangeHandler(_newState) {
   // does nothing
 }
 
 export async function makeWallet(
   E,
-  log,
-  host,
   zoe,
   registrar,
   pursesStateChangeHandler = mockStateChangeHandler,
   inboxStateChangeHandler = mockStateChangeHandler,
 ) {
-  // Map of purses in the wallet by pet name. Assume immutable.
-  const nameToPurse = new Map();
-  const nameToAssayId = new Map();
+  const petnameToPurse = makeStore();
+  const petnameToAssay = makeStore();
```

Comment:
Is this purse or assay petname?

Reply:
Good point, I can specify. It's the assay petname to the assay.